.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
```

Comment:
Maybe instead `MF-XXX` we can use `PYSDK-XXX`, to distiguish it from MF core.

Reply:
Added PYSDK